portal/src/graphql/adminapi/UserDetailsConnectedIdentities.module.scss

```diff
@@ -0,0 +1,42 @@
+.root {
+  margin-top: 25px;
+  margin-left: 10px;
+  width: 640px;
+}
+
+.header {
+  font-size: 14px;
+}
+
+.cellInfo {
+  display: flex;
+  padding: 5px 0 3px;
+}
+
+.cellIcon {
+  padding: 3px;
+}
+
+.cellDetail {
+  padding: 0 10px;
+  font-weight: 300;
+  font-size: 13px;
+}
+
+.cellName {
+  padding-bottom: 3px;
+}
+
+.cellDesc {
+  color: #888888;
+}
+
+.removeButton {
+  margin-right: 3px;
+  background-color: #ff0000;
+  border: 1px solid #990000;
+  &:hover {
+    background-color: #e80000;
+    border: 1px solid #990000;
+  }
```

Comment:
Can we introduce our Destructive button with a custom theme?

In `src/theme.tsx`, we have two themes. Let's add a new theme where the primary color is red.

And then we can add a component DestructiveButton which is basically a PrimaryButton with theme set to the new theme

```typescript
function DestructiveButton(props) {
  return <PrimaryButton theme={destructiveTheme} {...props} />
}
```

The theme generator is at https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/theming-designer/index.html

Reply:
Nice, didn't realize that there is a theme generator